src/Notifier.js

```diff
@@ -67,26 +69,17 @@ var Notifier = {
             ev.sender, 40, 40, 'crop'
         ) : null;
 
-        var notification = new global.Notification(
```

Comment:
how about the test at line 41 `global.Notification || global.Notification.permission != 'granted'` - that sounds like it should be in the Platform class?